src/provider/application.js

```diff
@@ -30,9 +30,12 @@ class Application extends EventEmitter {
     this.verifyChallenge = this.verifyChallenge.bind(this);
     this.emitError = this.emitError.bind(this);
     this.unload = this.unload.bind(this);
+    const self = this;
 
-    // Resize for slow loading images
-    document.body.addEventListener('load', this.imageRequestResize.bind(this), true);
+    // Resize for slow loading images when DOM is ready
+    document.addEventListener('DOMContentLoaded', () => {
```

Comment:
this works, but I think i prefer just setting the listener on the document which is always guaranteed to be present

Reply:
Also, I'm guessing in this scenario (where body isn't present) there is also a chance that mutation observer code below will break. We should probably fix that as well...
```js
 observer.observe(
            document.body, // This will be null if SDK is in head
            { attributes: true, childList: true, characterData: true, subtree: true }
);
```

Reply:
the observer probably shouldn't be attached until the DOMReady event fires

Reply:
Lets reoslve the mutation observer issue in a different pull request.

Reply:
Modified to add event listener on document